internal/cmd/version.go

```diff
@@ -20,7 +21,11 @@ import (
 )
 
 func versionCmdFunc(cmd *cobra.Command, _ []string) error {
-	if !term.IsTerminal(int(os.Stdout.Fd())) {
+	intFd, err := safecast.ToInt(uint(os.Stdout.Fd()))
+	if err != nil {
+		return err
+	}
+	if !term.IsTerminal(intFd) {
```

Comment:
is there a way to ignore the casting just for this one? 

I'm not sure why the lib is written to take an int from a uintptr but this is an `x` pkg: https://pkg.go.dev/golang.org/x/term#section-readme

Reply:
Yeah, that was making me scratch my head as well.

Reply:
https://github.com/golang/go/issues/69451

I'm not actually sure that we can ignore it, because I'm not actually sure this works on windows :thinking: